### PR TITLE
gh-251: Adding the submit and request-edit endpoints

### LINKED ISSUE #251: Enable submitedLock and unlock

### PR DESCRIPTION
**Pull request by dinukadesilva**
This fixes #251

### CODE REVIEW

File: api/TallySheetApi.py [hunk 1]
```diff
@@ -25,7 +25,8 @@ def getAll(electionId=None, areaId=None, tallySheetCode=None):
 
 @authorize(required_roles=ALL_ROLES)
 def get_by_id(tallySheetId):
-    tally_sheet = TallySheet.get_by_id(tallySheetId=tallySheetId)
+    tally_sheet = TallyS
+    heet.get_by_id(tallySheetId=tallySheetId)
```
Comment (callkalpa) [inline api/TallySheetApi.py]: Need to be in one line

File: swagger.yml [hunk 2]
```diff
@@ -3583,6 +3583,107 @@ paths:
...
+                $ref: '#/components/schemas/ApiResponse'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - submittedVersionId
+              properties:
+                lockedVersionId:
```
Comment (callkalpa) [inline swagger.yml]: should be "submittedVersionId"

**Review by callkalpa — CHANGES_REQUESTED** [review]

File: api/TallySheetApi.py (hunk 1 above)
Comment (dinukadesilva) [reply api/TallySheetApi.py]: Changed

File: swagger.yml (hunk 2 above)
Comment (dinukadesilva) [reply swagger.yml]: Changed

**Pull request merged**